client/components/fastlane/html/.nvmrc

```diff
@@ -0,0 +1 @@
+20.18.1
```

Comment:
I'm a fan of `nvmrc` files, but am cool with deleting this if there's opposition.